lib/Models/MagdaReference.ts

```diff
@@ -713,6 +714,13 @@ export default class MagdaReference extends AccessControlMixin(
     return undefined;
   }
 
+  @computed get cacheDuration(): string {
+    if (isDefined(super.cacheDuration)) {
+      return super.cacheDuration;
+    }
+    return "1d";
```

Comment:
The previous value of this was `0d`, does that matter?

Reply:
Given 0d was the previous default we should also set that to 0d here until we see how that interacts with the proxy, overriding the cache control behaviour of magda-gateway via our proxy wouldn't be ideal

>it's fine when we access it directly, because we've forced our gateway to set private cache headers, but i'm not sure how that interaction changes when we put requests to magda through our terriajs-server proxy

>i would hope it's fine and terriajs-server honours those kind of headers but best to double check

>actually i would say it's essential to double check, auth is new to terriajs and to terriajs-server een more., (in the sense that terriajs-server isn't designed for an auth world.)

Reply:
Related: https://github.com/TerriaJS/nsw-digital-twin/issues/345

Reply:
Yeah that was my bad, good pick up @nf-s

Reply:
Fixed